spring-cloud-generator/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>spring-cloud-gcp</artifactId>
+        <groupId>com.google.cloud</groupId>
+        <version>4.1.5-SNAPSHOT</version><!-- {x-version-update:spring-cloud-gcp:current} -->
+    </parent>
+
+    <groupId>com.google.api</groupId>
+    <artifactId>spring-cloud-generator</artifactId>
+    <name>Spring Code Generator for Google Cloud</name>
+    <description>Spring Code Generator for Google Cloud</description>
+
+    <properties>
+        <!-- [gapic-test]: for compiling test protos -->
+        <protobuf.version>3.21.12</protobuf.version>
```

Comment:
See comment below for the potential of extending from parent poms.

Reply:
@blakeli0 I wasn't able to find a straightforward way to inherit this version from `gapic-generator-java-bom`, since here `protobuf.version` is used as a property for configuring `protobuf-maven-plugin`, rather than as a dependency. 

https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/7d14fa45496bbe71ac252f8fd98ddb9489aec463/spring-cloud-generator/pom.xml#L53-L61

This plugin is only used for compiling test proto files for unit testing, so I’ve updated the inline comment to explain this for now. To follow-up on this PR, I’ll look into whether it can be removed altogether with a testlib jar published from `gapic-generator-java`.

Reply:
Makes sense, thanks for the explanation! 

I think renovate bot won't pick up on updating this version, since the only usage of the property is in a configuration literal and not a maven `<dependency>` block. Will keep an eye out and add an exclusion if one does come through.